pkg/apis/monitoring/v1/types_test.go

```diff
@@ -235,3 +235,75 @@ func TestValidateTLSConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateAuthorization(t *testing.T) {
+	for _, tc := range []struct {
+		config *Authorization
+		err    bool
+	}{
+		{
+			// minimal example
```

Comment:
Instead of adding comments, should we give each test case a name and use it as the first parameter in the `t.Run` function? This way, when there are failures, we will know which test case failed. We can also re-run failed cases only.